run.py

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I added a console_script, this can be invoked after installing the package with 'os-reststack-manager'